.github/workflows/build.yaml

```diff
@@ -245,14 +239,19 @@ jobs:
                   path: /tmp/cores/
                   # Cores are big; don't hold on to them too long.
                   retention-days: 5
-            - name: Uploading objdir for debugging
-              uses: actions/upload-artifact@v2
-              if: ${{ failure() && !env.ACT }}
-              with:
-                  name: crash-objdir-linux
-                  path: out/
-                  # objdirs are big; don't hold on to them too long.
-                  retention-days: 5
+            # OBJDIR on linux is > 10K files and takes more than 50 minutes to upload, usually
+            # having the job timeout. 
```

Comment:
If this is disabled, there's no point uploading the core either, right?  It's not particularly usable without the objdir...

That said, this did not take 50 minutes to upload when this job step was added.  Why is it taking that long now?

Reply:
It may make sense for us to clean output after every build step, to reduce the amount of items. This is probably some separate followup though: clean and likely even then only upload elf and so files for debugging, no need for all obj/ninja files.